qiskit/algorithms/state_fidelities/compute_uncompute.py

```diff
@@ -49,10 +49,25 @@ class ComputeUncompute(BaseStateFidelity):
 
     """
 
-    def __init__(self, sampler: BaseSampler, options: Options | None = None) -> None:
-        """
+    def __init__(
+        self,
+        sampler: BaseSampler,
+        average_local: bool = False,
```

Comment:
This new arguments needs to go after `options`, otherwise the behavior of calling the fidelity with positional arguments will change. Before this would work, but here it wouldn't anymore:
```python
fid = ComputeUncompute(my_sampler, my_options)
```